source/includes/config_repos/_01-materials.md.erb

```diff
@@ -0,0 +1,6 @@
+<%=
+describe_sub_object 'The pipeline material object', html_id: 'config-repo-material-object' do
+  type            'String',      'The type of a material.', one_of: %w(git svn hg p4 tfs)
+  attributes      'Object',      'The [attributes](#the-git-material-attributes) for each material type.'
```

Comment:
@varshavaradarajan -- we have specified all the types under type section, to show all the attributes, we have given an example of one of the materials(i.e. git)